exec/trino/trino.go

```diff
@@ -41,7 +40,7 @@ func NewTrinoExecutor(ctx context.Context, conf *TrinoConf) (*TrinoExecutor, err
 		conf.Port = 8080
 	}
 	password := url.QueryEscape(conf.Password)
-	dsn := fmt.Sprintf("http://%s:%s@%s:%d?catalog=%s&schema=%s", conf.User, password, conf.Host, conf.Port, conf.Catalog, conf.Schema)
+	dsn := fmt.Sprintf("http://%s:%s@%s:%d", conf.User, password, conf.Host, conf.Port)
```

Comment:
catalog no more part of DSN. User will be able set several catalogs for Trino integration